CI/travis.osx.install.sh

```diff
@@ -4,11 +4,46 @@ if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
   exit
 fi
 
-set -e
+set +e
 BREWS="boost cmake hunspell libzip libzzip lua51 pcre pkg-config qt5 yajl ccache pugixml"
```

Comment:
I am seeing error/warning message at the end of macOs builds possibly as part of the qtmacdeploy (?) operation where it is noting that `sqlite` is out of date and needs a later version to be brewed, I dunno if you want to add it here.

    ==> Installing wget
    ==> Downloading https://homebrew.bintray.com/bottles/wget-1.19.5.el_capitan.bott
    ==> Pouring wget-1.19.5.el_capitan.bottle.tar.gz
    🍺  /usr/local/Cellar/wget/1.19.5: 50 files, 3.7MB
    Error: sqlite 3.14.1 is already installed
    To upgrade to 3.23.1, run `brew upgrade sqlite`
    --2018-05-28 21:10:51--  https://raw.githubusercontent.com/aurelien-rainone/macdeployqtfix/master/macdeployqtfix.py
    Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.0.133, 151.101.64.133, 151.101.128.133, ...
    Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.0.133|:443... connected.
    HTTP request sent, awaiting response... 200 OK
    Length: 14597 (14K) [text/plain]
    Saving to: ‘macdeployqtfix.py’
    macdeployqtfix.py   100%[===================>]  14.25K  --.-KB/s    in 0.001s  
    2018-05-28 21:10:51 (15.3 MB/s) - ‘macdeployqtfix.py’ saved [14597/14597]
    Installing https://luarocks.org/luafilesystem-1.7.0-2.src.rock
    env MACOSX_DEPLOYMENT_TARGET=10.8 gcc -O2 -fPIC -I/usr/local/include/lua-5.1 -c src/lfs.c -o src/lfs.o
    env MACOSX_DEPLOYMENT_TARGET=10.8 gcc -bundle -undefined dynamic_lookup -all_load -o lfs.so -L/usr/local/lib src/lfs.o
    No existing manifest. Attempting to rebuild...
    luafilesystem 1.7.0-2 is now installed in /Users/travis/.luarocks (license: MIT/X11)
    Warning: Failed searching manifest: Failed fetching manifest for https://luarocks.org - Failed downloading https://luarocks.org/manifest - /Users/travis/.cache/luarocks/https___luarocks.org/manifest
    Warning: Failed searching manifest: Failed fetching manifest for https://raw.githubusercontent.com/rocks-moonscript-org/moonrocks-mirror/master/ - Failed downloading https://raw.githubusercontent.com/rocks-moonscript-org/moonrocks-mirror/master/manifest - /Users/travis/.cache/luarocks/https___raw.githubusercontent.com_rocks-moonscript-org_moonrocks-mirror_master_/manifest
    Warning: Failed searching manifest: Failed fetching manifest for http://luafr.org/moonrocks/ - Failed downloading http://luafr.org/moonrocks/manifest - /Users/travis/.cache/luarocks/http___luafr.org_moonrocks_/manifest
    Warning: Failed searching manifest: Failed fetching manifest for http://luarocks.logiceditor.com/rocks - Failed downloading http://luarocks.logiceditor.com/rocks/manifest - /Users/travis/.cache/luarocks/http___luarocks.logiceditor.com_rocks/manifest
    Error: No results matching query were found.
    /Users/travis/.travis/job_stages: line 57: shell_session_update: command not found

Reply:
I don't think we need a newer sqlite for the deploy and upgrading it would just take more time - I think this one is OK.